app/js/components/labOrderEntry/LabPanelFieldSet.jsx

```diff
@@ -1,27 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { panelData } from './labData';
 import '../../../css/grid.scss';
 
+
+const formatPanelName = (panelName) => {
+  const name = panelName;
+  const formattedName = name.split(' ')
+    .map(splittedName => splittedName[0].toUpperCase() + splittedName.substr(1).toLowerCase())
+    .join(' ')
+    .replace(/panel/i, '')
```

Comment:
where did the `i` come from?

Reply:
The `i` is a regular expression flag.
It is used to ignore case-sensitivity so that strings like `Panel`, `PaNel`, `paneL` e.t.c would all be a match.
You can find more details about regular expression flags [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Switching_words_in_a_string).

Reply:
I thought it was a path

Reply:
@IAMOTZ can't you still use the `text-transform` property here also ?

Reply:
Yes, I can.
I have updated the function.
Thanks, @rhenshaw56